doc/whatsnew/0/0.x.rst

```diff
@@ -658,7 +658,7 @@ Release date: 2006-08-10
 * started a reference user manual
```

Comment:
Not touching this file (unless it's a global sed) because it's a mix between Logilab's internal and bitbucket. Hard to tell, probably dead link anyway, and not very useful generally.